<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_details">
          <attribute name="title">
           <string>Detalles</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="verticalGroupBox_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Trabajadores:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="qworkers">
                <property name="minimumSize">
                 <size>
                  <width>132</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Horarios:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="qschedules">
                <property name="minimumSize">
                 <size>
                  <width>132</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="qdatefilter">
                <property name="text">
                 <string>Fechas:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="qdatesfilter">
                <property name="minimumSize">
                 <size>
                  <width>117</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Deshabilitado</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fecha</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Rango de fecha</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QStackedWidget" name="stackedWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>44</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="noFilter"/>
                <widget class="QWidget" name="singleDate">
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>Elija la fecha</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QDateEdit" name="qdate">
                        <property name="calendarPopup">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="datesRange">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QGridLayout" name="gridLayout_2">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_3">
                        <property name="layoutDirection">
                         <enum>Qt::LeftToRight</enum>
                        </property>
                        <property name="text">
                         <string>Desde:</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QDateEdit" name="qfromdate">
                        <property name="calendarPopup">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_2">
                        <property name="text">
                         <string>Hasta:</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QDateEdit" name="qtodate">
                        <property name="calendarPopup">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="qdatesrangebutton">
                      <property name="text">
                       <string>Fijar</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="qprint">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Imprimir</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_totals">
          <attribute name="title">
           <string>Totales</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="qcalculate">
                  <property name="text">
                   <string>Calcular Tiempo
Trabajado</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="printTotals">
                  <property name="text">
                   <string>Imprimir</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QTableView" name="tableView_total"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
